packages/ast-spec/tests/PunctuatorTokenToText.test.ts

```diff
@@ -0,0 +1,11 @@
+import type { PunctuationSyntaxKind } from 'typescript';
+
+import type { PunctuatorTokenToText } from '../src';
+
+// @ts-expect-error: purposely unused
+type _Test = {
+  readonly [T in PunctuationSyntaxKind]: PunctuatorTokenToText[T];
+  // If there are any PunctuationSyntaxKind members that don't have a
+  // corresponding PunctuatorTokenToText, then this line will error with
+  // "Type 'T' cannot be used to index type 'PunctuatorTokenToText'."
```

Comment:
Not really a helpful error message, but at least we know when we're not covering all `PunctuationSyntaxKind`'s 🤷‍♂️